### ISSUE TITLE
add share link to tagged photos

### ISSUE THREAD

**Issue by yekibud** [nextcloud/photos#360]
**Is your feature request related to a problem? Please describe.**
Currently, users can share photos only from albums.  Tagged photos are also groupings of photos which should have that same simple option.

**Describe the solution you'd like**
Add share link to tagged photos view.

**Describe alternatives you've considered**
There are none.

**Additional context**

I'd like to start work on this feature.  Any objections or suggestions?

**Comment by yekibud** [comment]
The scope of this issue is more specific, but could be considered a duplicate of https://github.com/nextcloud/photos/issues/120, and in any case, is blocked by larger structural issues related to the NC file-sharing system which is currently constrained by the WebDAV file-system paradigm.  So unfortunately I don't think I can move this forward myself without broader NC maintainer involvement.

**Comment by skjnldsv** [comment]
Hey! Thanks for your interest and your idea :)
I think this would have to be implemented into server directly, as we're using tags globally, not specifically for Photos :thinking: 

See https://github.com/nextcloud/server/issues/1291
